### DISCUSSION TITLE
Does this system is update 5.1 I want  learn Gas i don't want spend 1k hour learning for its basic usecase.

### DISCUSSION THREAD

**Discussion by ACMOIDRE** [Narxim/Narxim-GAS-Example#2] [Q&A] [answered]
thank you be for being generous  making simple system

**Comment by ACMOIDRE** [comment]
i heard they have changed few function changed in Abilitycomponent.h

**Comment by Narxim** [comment, marked as answer]
Hey, I'm really sorry for the late response, I wasn't even aware of this question since Github never gave any indication. >_< To my best knowledge the "5.0" branch as of this reply should compile in 5.0 - 5.3 at least. I'll be updating this to a 5.4 version shortly with some pretty major refactoring.
